sphinx/builders/latex/constants.py

```diff
@@ -198,6 +198,7 @@
         'polyglossia':  '',
         'babel':        '\\usepackage{babel}',
         'fontenc':      '\\usepackage{xeCJK}',
+        'fvset':        '\\fvset{fontsize=\\small,formatcom*=\\xeCJKVerbAddon}',
```

Comment:
I made wrong assumptions in https://github.com/sphinx-doc/sphinx/issues/9414#issuecomment-875705288. 
 - When used in the optional key-val argument of `Verbatim`-derived environment, for example `\begin{Verbatim}[...]`, `formatcom*` is preferable. 
 - When used in `\fvset{...}` in (a well structured) preamble, `formatcom` is enough. And `formatcom*` is harmless.

Reply:
ok, i changed to `formatcom=`